substrate/pkg/controller/substrate/cluster/addons/tekton.go

```diff
@@ -23,6 +23,69 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var rbac = `
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-dashboard
+  name: tekton-pipelines-executor
+rules:
+- apiGroups: ["kit.k8s.sh"]
+  resources: ["controlplanes", "dataplanes"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+- apiGroups: [""]
+  resources: ["serviceaccounts", "secrets", "namespaces", "nodes"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+- apiGroups: ["apiextensions.k8s.io"]
+  resources: ["customresourcedefinitions"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+- apiGroups: ["rbac.authorization.k8s.io"]
+  resources: ["clusterroles", "clusterrolebindings"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+- apiGroups: ["apps"]
+  resources: ["daemonsets"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+- apiGroups: ["karpenter.sh"]
+  resources: ["provisioners"]
+  verbs: ["get"]
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["certificatesigningrequests", "certificatesigningrequests/approval"]
+  verbs: ["get", "list", "watch", "create", "update", "delete", "patch"]
+- apiGroups: ["certificates.k8s.io"]
+  resources: ["signers"]
+  verbs: ["approve"]
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/component: dashboard
+    app.kubernetes.io/instance: default
+    app.kubernetes.io/part-of: tekton-dashboard
```

Comment:
For my understanding, why do we need these labels?
Why is this SA associated with dashboard?

Reply:
These are standard kubernetes labels https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/, copied from the tekton release.